srcts/src/components/tooltip.ts

```diff
@@ -28,64 +29,73 @@ type UpdateMessage = {
 
 type MessageData = ToggleMessage | UpdateMessage;
 
-type TooltipTypeWithTip = TooltipType & {
-  tip: HTMLElement | null;
-};
-
 export class BslibTooltip extends LightElement {
   static tagName = "bslib-tooltip";
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  _tooltip!: TooltipType;
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  _observer!: IntersectionObserver;
+  private tooltip!: TooltipType & { tip?: HTMLElement };
```

Comment:
Small nit, but I like `bsThing` as a naming convention to signal that `Thing` is a BS-provided instance

```suggestion
  private bsTooltip!: TooltipType & { tip?: HTMLElement };
```